src/components/trafficFlow.jsx

```diff
@@ -246,6 +324,16 @@ class TrafficFlow extends React.Component {
     this.setState({ displayOptions: displayOptions });
   }
 
+  physicsOptionsChanged = (physicsOptions) => {
+    this.setState({ currentGraph_physicsOptions: physicsOptions });
+    let currentGraph = this.state.currentGraph;
+    if (currentGraph == null) currentGraph = null;
+    Console.debug('TrafficFlow.jsx physicsOptionsChanged: ', physicsOptions, currentGraph, this.vizcReactComp);
```

Comment:
Are all these Console.debug messages just leftover from development? I don't see a need for these to be active in master...